stock_picking_delivery_slip_required/models/stock_picking.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2024 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    is_delivery_slip_required = fields.Boolean(
```

Comment:
Can you use `get_default` and update it so that it does not affect past transaction data?